libs/utils/include/utils/compiler.h

```diff
@@ -65,19 +65,21 @@
 #endif
 
 #define UTILS_NO_SANITIZE_THREAD
-#if defined(__has_feature)
-#    if __has_feature(thread_sanitizer)
-#        undef UTILS_NO_SANITIZE_THREAD
-#        define UTILS_NO_SANITIZE_THREAD __attribute__((no_sanitize("thread")))
-#    endif
+#if __has_feature(thread_sanitizer)
```

Comment:
I think we still need to check for `defined(__has_feature)`, otherwise MSVC complains.

Reply:
There is:
```
#ifndef __has_feature
#define __has_feature(x) 0
#endif
```
at the top of this file and existing uses of __has_feature w/o the check, e.g.:
```
#elif __has_feature(cxx_thread_local)
```

Reply:
Ah thanks I see that now.